osi_lane.proto

```diff
@@ -44,47 +44,64 @@ message Lane
     //
     // \image html line_approximation_error.png "Approximation error as green line."
     //
+    // \note Centerline is defined only for \c Lane::type = \c TYPE_DRIVING and one
+    // \c Lane::lane_pairing pair.
     repeated Vector3d center_line = 3;
 
-    // The boundary of the lane.
+    // Definition of the intended driving direction.
     //
-    // The lane boundaries describe the lane markings or other boundaries delimiting the lane as well as free lines that
-    // might be used e.g. in construction sites. The lane markings do not have to be parallel to the center_line, but
-    // they are the basis for calculating the width of the lane at any given distance.
+    // Defined and used for driving lanes.
+    // true means driving direction is according to ascending order of center line points.
```

Comment:
true means driving direction agrees with ascending storage order of the points defining the center_line.